distributed/bokeh/tests/test_worker_monitor.py

```diff
@@ -37,11 +39,17 @@ def test_resource_monitor_plot(s, a, b):
     assert len(source.data['times']) == 2
     assert len(source.data['memory-percent']) == 2
 
+    assert len(source.data['network-send']) == 2
+    assert source.data['network-send'][1] == ['null', 'null', 2**17]
```

Comment:
It looks like `resource_profile_update` is no longer in use.  Previously we used to show the cpu/memory use of *all* workers.  This has been replaced.  Presumably this test and the associated function can be removed.